### ISSUE TITLE
The use of "log" vs "ln", chapter 3 - Loss

### ISSUE THREAD

**Issue by minertom** [dvgodoy/PyTorchStepByStep#12]
I hope that this is not trivial. I was confused by it for a while so I thought that I should bring it up.

Normally, in engineering, when I see Log, without any base, it is assumed to be logarithm to the base 10.  In the following from chapter 3, Loss
`first_summation = torch.log(positive_pred).sum()`

Printing this first summation I noticed the value =   tensor(-0.1054)
I was going though the math and realized that this is not equal to log 10 of .9, which is -.045. 

Going to the pytorch documentation I saw that "log Returns a new tensor with the natural logarithm of the elements of input."

Of course, in the "From Logits to Probablilties" there is shown the relationship which "kind of" hints towards natural logarithms or log to the base e, but the whole confusion can be avoided by using the symbol "ln" as opposed to "log".

Do you agree?

Thank You
Tom

**Comment by dvgodoy** [comment]
Hi Tom,

I see you're moving quite fast :-)
Thanks for the feedback - you do have a point - I will put this in my to-do list for the final revision.

I guess each field has its own defaults for log... in ML, I've never seen log base 10 or log base 2, it is always natural log.
Since I come from a CS background (and it is always base 2 in information theory), it bugs me a bit to see 0.69 for log(2) instead of 1 bit :-)

Best,
Daniel

**Comment by minertom** [comment]
On Fri, Dec 4, 2020 at 11:36 AM Daniel Voigt Godoy <notifications@github.com>
wrote:

> Hi Tom,
>
> I see you're moving quite fast :-)
>

Well, this is not my first rodeo :-)
https://www.facebook.com/photo?fbid=1838315666307553&set=gm.2081015166079337
